.github/workflows/sdk-python-ci.yml

```
@@ -1,14 +1,12 @@
-name: Python SDK Tests
+name: sdk-python-ci
 
 on:
   push:
     branches:
       - main
-    paths:
-      - "python-sdk/**"
   pull_request:
     paths:
```

actionlint (1.7.7)

string should not be empty

(syntax-check)